jina/clients/mixin.py

```diff
@@ -521,6 +521,39 @@ async def post(
             else:
                 yield result
 
+    async def stream_doc(
+        self,
+        on: str,
+        inputs: Document,
+        parameters: Optional[Dict] = None,
+        return_type: Type[Document] = Document,
+        timeout: Optional[int] = None,
+        **kwargs,
+    ) -> AsyncGenerator[None, 'Document']:
+        """Send one document to a streaming endpoint and receive results asynchronisly.
```

Comment:
not only results asyncronously, it receives also a single Document

Reply:
I improved the wording to specify that it receives results document by document